contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -48,6 +67,7 @@ contract TrueCreditAgency is UpgradeableClaimable {
 
     function setRiskPremium(uint256 newRate) external onlyOwner {
         riskPremium = newRate;
+        // TODO keep separate premiums per pool, pass pool as an argument here
```

Comment:
TODO in this PR or in some next?

Reply:
some next

Reply:
I believe Roshan wanted a single premium, with no distinction between different pools. I would hold off on implementing separate premiums until/unless the feature is requested.